desktop/release-notes.md

```diff
@@ -28,7 +28,22 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/re
 
 {% include release-date.html date="2023-06-29" %}
```

Comment:
@lorenrh I guess we should update the version number and release date?

Reply:
Oh yes! my bad!